gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1046,6 +1093,54 @@ public static NetworkFilter notEquals(NetworkField field, boolean value) {
   }
 
   /**
+   * Class for filtering instance lists.
+   */
+  class InstanceFilter extends ListFilter {
+
+    private static final long serialVersionUID = 679008888882025686L;
+
+    private InstanceFilter(InstanceField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static InstanceFilter equals(InstanceField field, String value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method name and parameter number are suspiciously close to equals(Object)](https://www.codacy.com/app/mziccard/gcloud-java/file/2464634543/issues/source?bid=3140968&fileBranchId=3256314#l1113)